### PR TITLE
 Save patient sets to database when creating cross table.

### PR DESCRIPTION
**Pull request by gijskant**
Fixes [TMT-243](https://jira.thehyve.nl/browse/TMT-243).
Includes #275.

### CODE REVIEW

File: transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/MDStudySerializationHelper.groovy [hunk 1]
```diff
@@ -24,6 +24,7 @@ class MDStudySerializationHelper extends AbstractHalOrJsonSerializationHelper<St
                 id: study.id,
                 studyId: study.name,
                 bioExperimentId: study.bioExperimentId,
+                secureObjectToken: study.secureObjectToken,
```
Comment (ewelinagr) [inline transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/MDStudySerializationHelper.groovy]: Why is this needed to be exposed?

**Comment by ewelinagr** [comment]
This will break some of the e2e tests, but I can fix them in a separate PR

File: transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/MDStudySerializationHelper.groovy (hunk 1 above)
Comment (gijskant) [reply transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/MDStudySerializationHelper.groovy]: Mainly for data loading scripts to determine which studies are non-public, for loading roles into Keycloak (and which token to use).

**Comment by gijskant** [comment]
@ewelinagr For another ticket I am also fixing e2e tests, so I can fix this one as well.
